src/decorators/init-debounce/init-debounce.decorator.js

```diff
@@ -0,0 +1,13 @@
+const initDebounce = (delay) => {
```

Comment:
```suggestion
const debounce = (delay) => {
```
decorator name should be a noun